src/openrct2/ride/coaster/WoodenRollerCoaster.cpp

```diff
@@ -14548,300 +14746,310 @@ static void wooden_rc_track_booster(
         case TrackElemType::RightBankedUp25ToRightBankedFlat:
-            return wooden_rc_track_right_banked_25_deg_up_to_right_banked_flat;
+            return wooden_rc_track_right_banked_25_deg_up_to_right_banked_flat<isClassic>;
         case TrackElemType::LeftBankedFlatToLeftBankedDown25:
-            return wooden_rc_track_left_banked_flat_to_left_banked_25_deg_down;
+            return wooden_rc_track_left_banked_flat_to_left_banked_25_deg_down<isClassic>;
         case TrackElemType::RightBankedFlatToRightBankedDown25:
-            return wooden_rc_track_right_banked_flat_to_right_banked_25_deg_down;
+            return wooden_rc_track_right_banked_flat_to_right_banked_25_deg_down<isClassic>;
         case TrackElemType::LeftBankedDown25ToLeftBankedFlat:
-            return wooden_rc_track_left_banked_25_deg_down_to_left_banked_flat;
+            return wooden_rc_track_left_banked_25_deg_down_to_left_banked_flat<isClassic>;
         case TrackElemType::RightBankedDown25ToRightBankedFlat:
-            return wooden_rc_track_right_banked_25_deg_down_to_right_banked_flat;
+            return wooden_rc_track_right_banked_25_deg_down_to_right_banked_flat<isClassic>;
         case TrackElemType::FlatToLeftBankedUp25:
-            return wooden_rc_track_flat_to_left_banked_25_deg_up;
+            return wooden_rc_track_flat_to_left_banked_25_deg_up<isClassic>;
         case TrackElemType::FlatToRightBankedUp25:
-            return wooden_rc_track_flat_to_right_banked_25_deg_up;
+            return wooden_rc_track_flat_to_right_banked_25_deg_up<isClassic>;
         case TrackElemType::LeftBankedUp25ToFlat:
-            return wooden_rc_track_left_banked_25_deg_up_to_flat;
+            return wooden_rc_track_left_banked_25_deg_up_to_flat<isClassic>;
         case TrackElemType::RightBankedUp25ToFlat:
-            return wooden_rc_track_right_banked_25_deg_up_to_flat;
+            return wooden_rc_track_right_banked_25_deg_up_to_flat<isClassic>;
         case TrackElemType::FlatToLeftBankedDown25:
-            return wooden_rc_track_flat_to_left_banked_25_deg_down;
+            return wooden_rc_track_flat_to_left_banked_25_deg_down<isClassic>;
         case TrackElemType::FlatToRightBankedDown25:
-            return wooden_rc_track_flat_to_right_banked_25_deg_down;
+            return wooden_rc_track_flat_to_right_banked_25_deg_down<isClassic>;
         case TrackElemType::LeftBankedDown25ToFlat:
-            return wooden_rc_track_left_banked_25_deg_down_to_flat;
+            return wooden_rc_track_left_banked_25_deg_down_to_flat<isClassic>;
         case TrackElemType::RightBankedDown25ToFlat:
-            return wooden_rc_track_right_banked_25_deg_down_to_flat;
+            return wooden_rc_track_right_banked_25_deg_down_to_flat<isClassic>;
         case TrackElemType::Booster:
-            return wooden_rc_track_booster;
+            return wooden_rc_track_booster<isClassic>;
```

Comment:
I'm curious if all this stuff is even necessary. Couldn't this just be done by making the classic coastervall pieces it doesn't have itself?